frontend/src/components/verticalPodAutoscaler/List.tsx

```diff
@@ -0,0 +1,79 @@
+import { Link, Paper, Typography } from '@mui/material';
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import VPA from '../../lib/k8s/vpa';
+import { SectionBox } from '../common';
+import Empty from '../common/EmptyContent';
+import ResourceListView from '../common/Resource/ResourceListView';
+
+export default function VpaList() {
+  const { t } = useTranslation(['glossary', 'translation']);
+  const [vpaEnabled, setVpaEnabled] = useState<boolean | null>(null);
+
+  useEffect(() => {
+    const vpaStatus = async () => {
+      const enabled = await VPA.isEnabled();
+      setVpaEnabled(enabled);
+    };
+    vpaStatus();
+  }, []);
+
+  return (
+    <>
+      {vpaEnabled === null ? (
+        <SectionBox title={t('glossary|Vertical Pod Autoscalers')}>
+          <Paper variant="outlined">
+            <Empty>
+              <Typography style={{ textAlign: 'center' }}>
+                Checking if Vertical Pod Autoscaler is enabled...
```

Comment:
This needs to be translatable. And the elipsis character should be one single character (not three period characters).